Gemfile

```diff
@@ -5,8 +5,9 @@ gem 'sqlite3'
 gem 'puma'
 gem 'bcrypt'
 gem 'faker'
+gem 'sprockets-rails'
```

Comment:
Is this used?

Reply:
It was briefly, but that setup was over-complicating things. Will ✂️